data/126/021/452/3/1260214523.geojson

```diff
@@ -53,10 +55,14 @@
         }
     ],
     "wof:id":1260214523,
-    "wof:lastmodified":1566639591,
-    "wof:name":"Tomba Dougoula",
+    "wof:lastmodified":1701132353,
+    "wof:name":"Tomba Kanssa",
```

Comment:
When we replace the default `wof:name` with a different value, we want to also preserve the original name in a variants list. In my other PR I set the new default name to `name:eng_x_preferred` and store the old name into `name:eng_x_variant`. (You could argue `fra` instead... but there are no diacritical marks in these names, and WOF's default language is English, so if there is no French specified the downstream service should look for English, and if no English then use `wof:name`).